web/src/components/navbar/desktop/NavBarDesktop.tsx

```diff
@@ -0,0 +1,185 @@
+import { ButtonIcon } from "@/components/ButtonIcon";
+import { NavBarLogoOnly } from "@/components/navbar/NavBarLogoOnly";
+import { DesktopLoginButton } from "@/components/navbar/desktop/DesktopLoginButton";
+import { NavBarDesktopDropDown } from "@/components/navbar/desktop/NavBarDesktopDropDown";
+import { NavBarDesktopHomeLogo } from "@/components/navbar/desktop/NavBarDesktopHomeLogo";
+import { NavBarDesktopQuickLinks } from "@/components/navbar/desktop/NavBarDesktopQuickLinks";
+import { NavBarVerticalLineDivider } from "@/components/navbar/desktop/NavBarDesktopVerticalLineDivider";
+import { NavBarDesktopWrapper } from "@/components/navbar/desktop/NavBarDesktopWrapper";
+import {
+  AddBusinessItem,
+  GetStartedMenuItem,
+  LogoutMenuItem,
+  MyNjMenuItem,
+  ProfileMenuItem,
+  RegisterMenuItem,
+} from "@/components/navbar/shared-submenu-components";
+import { Icon } from "@/components/njwds/Icon";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { getBusinessIconColor } from "@/lib/domain-logic/getBusinessIconColor";
+import { getNavBarBusinessTitle } from "@/lib/domain-logic/getNavBarBusinessTitle";
+import { orderBusinessIdsByDateCreated } from "@/lib/domain-logic/orderBusinessIdsByDateCreated";
+import React, { ReactElement, useEffect, useRef, useState } from "react";
+
+interface Props {
+  previousBusinessId?: string | undefined;
+  isLanding: boolean | undefined;
+  logoOnlyType?: "NAVIGATOR_LOGO" | "NAVIGATOR_MYNJ_LOGO" | undefined;
+  currentlyOnboarding: boolean | undefined;
+  isAuthenticated: boolean;
+}
+
+export const NavBarDesktop = (props: Props): ReactElement => {
+  const { business, userData } = useUserData();
```

Comment:
dust - if we pass in userdata as a prop, this component can be a dummy UI component

Reply:
Also userData?